pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7222,6 +7583,39 @@ def solve_model(model, options=None):
                         model_h, options={"limits/gap": 0.3}, tee=True
                     )
 
+        elif model.config.hydraulics == Hydraulics.co_optimize_linearized:
+            # Currently, this method is supported for only MINLP solvers accessed through GAMS.
+            # The default solver is Baron and is set to find the first feasible solution.
+            # If the user has SCIP, then the feasible solution from BARON is passed to SCIP, which is then solved to a gap of 0.3.
+            # If the user do not have these solvers installed then it limits the use of co_optimize method at this time.
+            # Ongoing work is geared to address this limitation and will soon be updated here.
+
+            # Adding temporary variable bounds until the bounding method is implemented for the following Vars
+            model_h.v_F_Piped.setub(1050)
+            model_h.hydraulics.v_Pressure.setub(3.5e6)
+
+            try:
+                # solver = SolverFactory("gams")
+                solver = SolverFactory("gurobi")
+            except:
+                print(
+                    "Either GAMS or a license to BARON was not found. Please add GAMS to the path. If you do not have GAMS or BARON, \
+                      please continue to use the post-process method for hydraulics at this time"
+                )
+            else:
+                results_gurobi = solver.solve(model_h, tee=True, keepfiles=True)
+
+                results_2 = results_gurobi
+                # average_pumphead = 0
```

Comment:
Same question as above - is there a particular reason for keeping this block of commented code, or can it be removed instead?